src/app/pages/MediaArticlePage/SecondaryColumn.jsx

```diff
@@ -1,43 +1,39 @@
 /** @jsxRuntime classic */
 /** @jsx jsx */
-
 import path from 'ramda/src/path';
-import { jsx, useTheme } from '@emotion/react';
-
+import { jsx } from '@emotion/react';
 import { articleDataPropTypes } from '#models/propTypes/article';
-
-import FeaturesAnalysis from '#containers/CpsFeaturesAnalysis';
-import TopStoriesSection from './PagePromoSections/TopStoriesSection';
+import styled from '@emotion/styled';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '#psammead/gel-foundations/src/spacings';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '#psammead/gel-foundations/src/breakpoints';
+import LatestMediaSection from './PagePromoSections/LastestMediaSection';
 import styles from './MediaArticlePage.styles';
 
+const ResponsiveComponentWrapper = styled.div`
+  margin-bottom: ${GEL_SPACING_TRPL};
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING};
+    padding: ${GEL_SPACING_DBL};
+  }
+`;
```

Comment:
Could this styling be moved into the `styles.ts` file? Would save having to import the GEL spacings and breakpoints as they are available on the Theme.